charts/prow/values.yaml

```diff
@@ -0,0 +1,167 @@
+# Default values for prow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+githubBotAccount: samsungbot
+trustedOrgs: samsung-cnct, kubernetes
+githubHookConsumerAddress: "https://prow.cnct.io/hook"
+jenkinsaddress: ""
+repos: 
+  - samsung-cnct/prow-chart
+#  - samsung-cnct/lbex 
```

Comment:
Those are random repos that I used for testing purposes.  The chart is designed to take repos as an argument, and it will automatically configure the hooks into prow.  We should replace with real k8s repos.  Can you suggest a few that would be good to test?